package/network/services/odhcpd/Makefile

```diff
@@ -38,48 +37,32 @@ define Package/odhcpd/default/description
  requirements for IPv6 home routers.
 endef
 
-define Package/odhcpd/default/config
-menu "Configuration"
-	depends on PACKAGE_$(1)
-
-config PACKAGE_odhcpd_$(2)_ext_cer_id
-	int
-	default 0
-	prompt "CER-ID Extension ID (0 = disabled)"
-endmenu
-endef
-
 define Package/odhcpd
-  $(call Package/odhcpd/default)
-  TITLE += and DHCPv4 server
+  $(Package/odhcpd/default)
+  TITLE += with DHCPv4 support
   VARIANT:=full
 endef
 
-Package/odhcpd/config=$(call Package/odhcpd/default/config,odhcpd,full)
-
 define Package/odhcpd/description
- $(call Package/odhcpd/default/description)
+ $(Package/odhcpd/default/description)
 
- This is a variant providing server services for DHCPv4, RA, stateless and
- stateful DHCPv6,  prefix delegation and can be used to relay RA, DHCPv6 and
- NDP between routed (non-bridged) interfaces in case no delegated prefixes
- are available.
+ This is a variant with support for RA, DHCPv4 and DHCPv6. It can also be used
+ to relay RA, DHCPv6 and NDP messages between routed (non-bridged) interfaces
+ in case no delegated prefixes are available.
 endef
 
 define Package/odhcpd-ipv6only
-  $(call Package/odhcpd/default)
+  $(Package/odhcpd/default)
   VARIANT:=ipv6only
   DEPENDS+= @IPV6
 endef
 
-Package/odhcpd-ipv6only/config=$(call Package/odhcpd/default/config,odhcpd-ipv6only,ipv6only)
-
 define Package/odhcpd-ipv6only/description
- $(call Package/odhcpd/default/description)
+ $(Package/odhcpd/default/description)
 
- This is a variant providing server services for RA, stateless and stateful
- DHCPv6,  prefix delegation and can be used to relay RA, DHCPv6 and NDP between
- routed (non-bridged) interfaces in case no delegated prefixes are available.
+ This is a variant with support for RA and DHCPv6. It can also be used to
+ relay RA, DHCPv6 and NDP messages between routed (non-bridged) interfaces
+ in case no delegated prefixes are available.
```

Comment:
Can we accent more the absence of IPv4 here? Its implicit absence is often missed by part-time eejits like me.

Reply:
Few people read the description...I think it would be even better to emphasise the lack of IPv4 in the title in that case? (Which shows up in the make menuconfig UI)